utilities/fill_counties.py

```diff
@@ -34,10 +36,11 @@ def to_row(county):
     uf = county["Nome_UF"]
     try:
```

Comment:
as `county_polygon` doesn't raise error anymore, you can remove the try/except, and just keep:

```python
geojson = county_polygon(initials[uf], county_code)
print(county_code, name, if)
```